### PR TITLE
AM2R: Make DB changes for the A2 Water Turbine

### PR DESCRIPTION
**Pull request by Miepee**

### CODE REVIEW

File: randovania/games/am2r/json_data/header.txt [hunk 1]
```diff
@@ -324,7 +324,7 @@ Dock Weaknesses
           Can Use Power Bombs
 
 
-  * Water Turbine Door
+  * Hydro Station Water Turbine
       Open:
           Trivial
```
Comment (JeffGainsNGames) [inline randovania/games/am2r/json_data/header.txt:329]: Don't the the turbines require missiles/bombs?

Comment (Miepee) [reply randovania/games/am2r/json_data/header.txt:329]: It's trivial to open. It's lock however, is not. 
Scroll down a little bit further, and you'll see:
``` 
Lock type: FRONT_BLAST_BACK_FREE_UNLOCK
Destroy Turbine
```

**Review by JeffGainsNGames — APPROVED** [review]

File: randovania/games/am2r/json_data/Distribution Center.txt [hunk 2]
```diff
@@ -1394,7 +1394,7 @@ Extra - minimap_data: [{'x': 66, 'y': 49}, {'x': 66, 'y': 50}, {'x': 66, 'y': 51
 
 > Door to Serris Arena; Heals? False
   * Layers: default
-  * Missile Door to Serris Arena/Door to Serris Arena Access; Dock Lock Rando incompatible with: Spider Ball Door, Screw Attack Door
+  * Missile Door to Serris Arena/Door to Serris Arena Access; Dock Lock Rando incompatible with: Spider Ball Door, Screw Attack Door, Hydro Station Water Turbine
```
Comment (XenoWars) [inline randovania/games/am2r/json_data/Distribution Center.txt:1397]: Spider Ball exclusion on all these doors? for what reason? iirc, you can use Spider underwater.

Comment (Miepee) [reply randovania/games/am2r/json_data/Distribution Center.txt:1397]: Spider doesnt work underwater without gravity. It only works in the emp area without gravity

**Pull request merged**